apps/website/docs/pages/commands/overview.md

```diff
@@ -13,12 +13,14 @@ Sparo has four kinds of subcommands:
    - `sparo clone`
```

Comment:
The text above says `There are four enhanced commands:` -- seems like now there are 5